### ISSUE TITLE
[amplify_analytics_pinpoint] Upgrade `package_info_plus` dependency.

### ISSUE THREAD

**Issue by asaarnak** [aws-amplify/amplify-flutter#5255]
Increase `package_info_plus` from `^6.0.0` to `^8.0.0`

```
Because amplify_auth_cognito >=2.3.0 depends on amplify_analytics_pinpoint >=2.3.0 <2.4.0 which depends on package_info_plus ^6.0.0, amplify_auth_cognito >=2.3.0 requires package_info_plus ^6.0.0.
So, because my_app depends on both amplify_auth_cognito ^2.3.0 and package_info_plus ^8.0.0, version solving failed.
```

**Comment by khatruong2009** [comment]
Hi @asaarnak, thank you for submitting this request. We will take a look and get back to you with any updates.

**Comment by NikaHsn** [comment]
I'm going to close this issue as the fix is release in [2.4.0](https://github.com/aws-amplify/amplify-flutter/releases/tag/v2.4.0) version. If you see otherwise or have additional questions, please open a new issue.
